web/tests/unit/helpers/or-test.js

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
As mentioned in above comment, this test may not be required; albeit an acceptance/integration test may be written for the above UI change.